.vscode/settings.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
The .vscode folder probably shouldn't be committed

Reply:
ok